components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -561,19 +561,22 @@ - (void)viewWillAppear:(BOOL)animated {
 }
 
 - (void)setupLayout {
-  // Layout the clipping view and the scroll view.
-  if (self.currentlyFullscreen) {
-    CGRect scrollViewFrame = self.presentingViewBounds;
-    self.scrollView.frame = scrollViewFrame;
-  } else {
-    CGRect scrollViewFrame = self.presentingViewBounds;
+  // Layout the scroll view.
+  CGRect scrollViewFrame = self.presentingViewBounds;
+  if (!self.currentlyFullscreen) {
     if (self.animatingPresentation) {
       CGFloat heightSurplusForSpringAnimationOvershooting =
           self.presentingViewBounds.size.height / 2;
       scrollViewFrame.size.height += heightSurplusForSpringAnimationOvershooting;
     }
-    self.scrollView.frame = scrollViewFrame;
+
+    // Adjust y-position of origin to account for non-fullscreen presentation styles.
+    CGFloat yOffset = CGRectGetHeight(self.view.frame) - CGRectGetHeight(self.presentingViewBounds);
```

Comment:
the before video makes me think that this was a safe area issue. Can we double check the non safe area use case to make sure we are not pushing that down too far?

Reply:
That was my initial suspicion as well, but I am pretty confident that this is due to how the scrollView frame was being set prior to this PR. 

I have sanity checked this on non-notched device and added screenshot to [original bug](https://b.corp.google.com/issues/148977218). 😊

Reply:
cool thanks for checking.

Reply:
Sure thing good call on double-checking this!